### PR TITLE
Add Python 3.8 to tox.ini files

### PR DESCRIPTION
**Pull request by gsnedders**
This presumably won't run on CI, but at least means `tox` does what you want if you have Py3.8 installed locally.

### CODE REVIEW

**Review by jgraham — APPROVED** [review]

**Comment by gsnedders** [comment]
@jgraham TC failure now is bogus. Do we want to land this with all these extra jobs or not?

**Comment by jgraham** [comment]
General I think this is OK (I haven't reviewed the details), but can we call the jobs `Python 3 minimum (3.6)` and `Python 3 maximum (3.8)` or something instead. Having `3` and `3.8` might suggest it's like `3.0` or some nonsense.

**Comment by gsnedders** [comment]
@jgraham I've just renamed them to "3.6" to be explicit

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -145,6 +210,53 @@ jobs:
...
+  pool:
+    vmImage: 'macOS-10.14'
+  steps:
+  # full checkout required
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.6.x'
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '2.7.x'
```
Comment (foolip) [inline .azure-pipelines.yml]: Was it a copy-paste error to install 3.6 and then 2.7 directly after?

File: .azure-pipelines.yml [hunk 2]
```diff
@@ -145,6 +210,53 @@ jobs:
...
+    inputs:
+      versionSpec: '3.8.x'
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '2.7.x'
+  - template: tools/ci/azure/install_chrome.yml
+  - template: tools/ci/azure/install_firefox.yml
+  - template: tools/ci/azure/update_hosts.yml
+  - template: tools/ci/azure/update_manifest.yml
+  - template: tools/ci/azure/tox_pytest.yml
```
Comment (foolip) [inline .azure-pipelines.yml]: This is the other case like https://github.com/web-platform-tests/wpt/pull/20725/files#r377569491, where 3.8 installed followed by 2.7 directly after.

File: .azure-pipelines.yml (hunk 1 above)
Comment (foolip) [reply .azure-pipelines.yml]: I figured it out, will make it clearer in https://github.com/web-platform-tests/wpt/pull/21724.

**Pull request merged**